nss-cohort-40/banagzon-workforce-mgt-lumberjacks
# De add department

# Description
Added the ability for the user to add a new department by allowing them to navigate to a form from the list view, entering in the department info, and submitting it.
 Fixes #7 
 ## Type of change
 Please delete options that are not relevant.
 # Testing Instructions
Test by going to the departments list, clicking on the new departments button, and filling out the form.  You should be navigated back to the list and be able to see the new department at the bottom.  You can also just go directly to `departments/forms` to go to the form.
 # Checklist:
